.reaction/docker/scripts/build-meteor.sh

```diff
@@ -1,30 +1,35 @@
 #!/bin/bash
 
 #
-# builds a production meteor bundle directory
+# builds a production meteor bundle
 #
 set -e
 
-printf "\n[-] Building Meteor application...\n\n"
-
-: ${APP_BUNDLE_DIR:="/var/www"}
-: ${BUILD_SCRIPTS_DIR:="/opt/reaction"}
-
 cd $APP_SOURCE_DIR
 
 # Customize packages
-bash $BUILD_SCRIPTS_DIR/build-packages.sh
+$BUILD_SCRIPTS_DIR/build-packages.sh
 
 # Generate plugin import files
-bash $BUILD_SCRIPTS_DIR/plugin-loader.sh
+printf "\n[-] Running Reaction plugin loader...\n\n"
+$BUILD_SCRIPTS_DIR/plugin-loader.sh
 
 # Install app deps
+printf "\n[-] Running npm install...\n\n"
 meteor npm install
 
-# build the source
+# build the production bundle
+printf "\n[-] Building Meteor application...\n\n"
 mkdir -p $APP_BUNDLE_DIR
-meteor build --directory $APP_BUNDLE_DIR
-cd $APP_BUNDLE_DIR/bundle/programs/server/ && meteor npm install --production
+meteor build --unsafe-perm --directory $APP_BUNDLE_DIR
```

Comment:
Yes, the entire build process is done by root.  Then the resulting built app is `chown`'d to the non-root `node` user ([here](https://github.com/reactioncommerce/reaction/blob/master/.reaction/docker/scripts/build-meteor.sh#L34-L35)).  When the container gets started, the `node` user is runs the process.  All of the security benefits, none of the permissions hassles.